include/iso15118/session/feedback.hpp

```diff
@@ -33,14 +38,18 @@ struct DcMaximumLimits {
     float power{NAN};
 };
 
-using PresentVoltage = message_20::datatypes::RationalNumber;
+using PresentVoltage = dt::RationalNumber;
 using MeterInfoRequested = bool;
-using DcReqControlMode = std::variant<
-    message_20::datatypes::Scheduled_DC_CLReqControlMode, message_20::datatypes::BPT_Scheduled_DC_CLReqControlMode,
-    message_20::datatypes::Dynamic_DC_CLReqControlMode, message_20::datatypes::BPT_Dynamic_DC_CLReqControlMode>;
+using DcReqControlMode = std::variant<dt::Scheduled_DC_CLReqControlMode, dt::BPT_Scheduled_DC_CLReqControlMode,
+                                      dt::Dynamic_DC_CLReqControlMode, dt::BPT_Dynamic_DC_CLReqControlMode>;
+
+using DcChargeLoopReq = std::variant<DcReqControlMode, dt::DisplayParameters, PresentVoltage, MeterInfoRequested>;
+
+// TODO(ioan): preparation for AC limits
+using EVSE_TransferLimits = std::variant<d20::DcTransferLimits>;
 
-using DcChargeLoopReq =
-    std::variant<DcReqControlMode, message_20::datatypes::DisplayParameters, PresentVoltage, MeterInfoRequested>;
+using EV_TransferLimits = std::variant<dt::DC_CPDReqEnergyTransferMode, dt::BPT_DC_CPDReqEnergyTransferMode>;
+using EV_SEControlMode = std::variant<dt::Dynamic_SEReqControlMode, dt::Scheduled_SEReqControlMode>;
```

Comment:
Renaming to `EvSEControlMode`. I know this is not ideal because of `Evse` and `EvSE`, but with our naming convention this should be `EvSE`